spec/spec_app/config/environment.rb

```diff
@@ -29,6 +29,15 @@ def call(env)
   application.config.praxis.validate_responses = true
   application.config.praxis.show_exceptions = true
 
+  # Silly callback code pieces to test that the deferred callbacks work even for sub-stages
+  application.bootloader.after :app, :models do
+    PersonModel.identity(:other_id)
+  end
+  application.bootloader.after :app do
+    raise "After sub-stage hooks not working!" unless PersonModel.identities.include? :other_id
```

Comment:
Using this as a "test" seems about as wrong as wrong can get. Not really sure how to actually do it cleanly otherwise. 

Maybe put a big "FIXME" above that so we can at least feel better about it?